static/chat.css

```
@@ -206,13 +206,27 @@ body[data-page="chat"] [data-message] {
   padding: 12px 14px;
   border-radius: 16px;
   background: rgba(255, 255, 255, 0.06);
+  display: grid;
+  gap: 8px;
   font-size: 14px;
   line-height: 1.6;
-  white-space: pre-wrap;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)